packages/lesswrong/components/Layout.jsx

```diff
@@ -24,7 +24,7 @@ const styles = theme => ({
   main: {
     margin: '50px auto 15px auto',
     [theme.breakpoints.down('sm')]: {
-      marginTop: 0,
+      marginTop: -16,
```

Comment:
Did you actually check that this doesn't break stuff on other pages? This feels relatively risky.

Reply:
checked a couple pages, including the community page which seemed most likely to have weird behavior, which still looked reasonable to me.